flask_wiki/views.py

```diff
@@ -151,12 +151,24 @@ def preview():
     data['html'], data['body'], data['meta'], data['toc'] = processor.process()
     return data['html']
 
-@blueprint.route('/delete/<path:url>/', methods=['GET', 'POST'])
+@blueprint.route('/page/delete/<path:url>')
 @can_edit_permission
-def delete(url):
+def delete_page(url):
+    print('asjdhaskj', url)
     current_wiki.delete(url)
```

Comment:
you can check the return value to change the flash message depending if the page has been removed or not.